### PR TITLE
Calendar events

### PR DESCRIPTION
**Pull request by AhmadMoolla**
Lecturers and students can see their upcoming events on the calendar dashboard view.

There isn't functionality for a student to join a consultation yet though.

### CODE REVIEW

File: public/clientScripts/calendarControl.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+'use strict'
```
Comment (MarantosGeorge) [inline public/clientScripts/calendarControl.js:1]: Good practice well done

**Review by MarantosGeorge — CHANGES_REQUESTED** [review]
Line 85 in studentDashboard.js remove the commented code
Otherwise everything else looks good

File: public/clientScripts/studentDashboard.js [hunk 2]
```diff
@@ -17,73 +17,72 @@ async function fetchStudentDetails () {
...
+    })
 
-        tableBody.appendChild(row);
-    });
+    tableBody.appendChild(row)
+  })
 }
 
 // Call the function to create the table with the sample data
 // createTable(dataArray);
```
[the remark this answers is not in the document]
Comment (AhmadMoolla) [reply public/clientScripts/studentDashboard.js]: Commented code was removed

**Review by MarantosGeorge — APPROVED** [review]
Code follows correct principles according to code review guide

**Pull request merged**